src/test/java/w15c2/tusk/testutil/TestTask.java

```diff
@@ -0,0 +1,139 @@
+package w15c2.tusk.testutil;
+
+import w15c2.tusk.model.task.Description;
+import w15c2.tusk.model.task.Task;
+
+/**
+ * A mutable task object. For testing only.
+ */
+public class TestTask extends Task {
+
+	private Description description;
+	private boolean pinned;
+	private boolean completed;
+	
+	public TestTask(){
+	}
+	
+    public TestTask(Description description) {
+		super(description);
+	}
+
+    public TestTask(TestTask task2) {
+		this.description = task2.getDescription();
+		this.pinned = task2.isPinned();
+		this.completed = task2.isCompleted();
+	}
+
+	public void setDescription(Description description) {
+        this.description = description;
+    }
+
+    public void setPinned(boolean pinned) {
+        this.pinned = pinned;
+    }
+
+    public void setCompleted(boolean completed) {
+        this.completed = completed;
+    }
+
+    @Override
+    public Description getDescription() {
+        return description;
+    }
+
+    @Override
+    public boolean isCompleted() {
+        return completed;
+    }
+
+    @Override
+    public boolean isPinned() {
+        return pinned;
+    }
+
+    @Override
+    public String toString() {
+		return description.toString();
+    }
+
+    public String getAddCommand() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("add " + this.getDescription().getContent() + " ");
+        sb.append("a/" + this.isPinned() + " ");
+        sb.append("e/" + this.isCompleted() + " ");
+        return sb.toString();
```

Comment:
this is still the same as address book but i have the modified version in master already so i'll see how to integrate these classes